topics/janis/index.md

```diff
@@ -0,0 +1,19 @@
+---
+aliases: janis-commerce,janis-ecommerce
+created_by: Francisco Mato, Lisandro Aubert
+display_name: Janis
+github_url: https://github.com/janis-commerce
+logo: janis.png
+related: saas, platform, ecommerce
+released: 2014
+short_description: Janis transforms the B2C, B2B and marketplace operation, transforming ecommerce channel and omnichannel sales.
+topic: janis
+url: https://www.janis.im
+---
+Janis combine the power of a WMS, OMS & TMS in an Intelligent Distribution Manager, optimizing each resource and reducing operating costs with complete traceability of the operation.
+
+Janis has some of the largest businesses in Latin America and Spain as clients, managing millions of orders each year.
+
+Janis eCommerce is a high-growth Cloud based SaaS, supporting the most demanding operations in the retail, logistics, fashion, pharmaceutical, fresh and frozen food industries.
+
+Combine the fullfilment operation, picking, packing, catalogue, stock, internal movements, pricing policy, logistics services, last mile, all on a single platform: Janis.
```

Comment:
@cgobat I haven't worked there for a month anymore, I'm in another company. Maybe I didn't update my Linkedin profile, I'll check it out. If it is a conflict for you, we close the PR and that's it, it's not a problem for me.

@kenyonj Next weekend I will dedicate some time to improve all the changes that you have suggested, you are right, perhaps I focused on giving a friendlier base to the topics and not boring you with so many technical data, but even so, I agree with you. When I create a space in my agenda, I correct all together.

Reply:
Okay, all good with me. Not saying the PR is invalid because of it, I was just speculating that might have been part of the reason your description had a "marketing" tone to it.